pkg/controller/razeedeployment/razeedeployment_controller.go

```diff
@@ -1436,15 +1436,17 @@ func (r *ReconcileRazeeDeployment) fullUninstall(
 		if err == nil {
 			for _, cr := range customResourceList.Items {
 				reqLogger.Info("Deleteing custom resource", "custom resource", cr)
-
-				err := r.client.Delete(context.TODO(), &cr, client.PropagationPolicy(deletePolicy))
+				err := r.client.Delete(context.TODO(), &cr)
 				if err != nil && !errors.IsNotFound(err) {
 					reqLogger.Error(err, "could not delete custom resource", "custom resource", cr)
 				}
 			}
 		}
 	}
 
+	// sleep 5 seconds to let custom resource deletion complete
+	time.Sleep(time.Second * 5)
```

Comment:
Actually - this is not what we want to do. We should add this code to line 1442.

```
return reconcile.Result{requeueAfter: time.Second * 5}, nil
```

Reply:
Talked it over with Chen, she did it that way over a comment I made about worrying about dependent finalizers not finishing appropriately and re-queue forever since this is the finalizer. Traditionally we requeue this way, but since this is a finalizer we'll just use sleep.